### ISSUE TITLE
Plugin error: Cannot read property 'name' of undefined.

### ISSUE THREAD

**Issue by relayer464** [TiagoSilvaPereira/vemto-issues#303]
Hi Tiago, 

Several months ago you helped me figure out how to use a custom plugin to generate new project pages.  For example:

```
let models = vemto.getProjectModels()

models.forEach(model => {
    let options = {}
    options.data = {model}

    vemto.renderTemplate('/files/ListView.vemtl', '/resources/views/app/' + model.table + '/list.blade.php', options)
})
```

This code was working fine.  However, a few days ago I was making minor updates to the plugin and now I get an error that reads:

> TypeError: Cannot read property 'name' of undefined.  
>  

Here is a screenshot of the error:

![](https://user-images.githubusercontent.com/20197966/164777391-60f4a26d-f02d-4646-842f-8d65777911f5.png)

Basically all I am trying to do is create an alternate "Index" page view.  I've tried removing my custom code and replacing it with the same code that Vemto uses by default for the Index view but I still get the error.  The only way I can get the files to generate is if I use pure HTML and no Vemto template references.   

Any suggestions you may have on resolving this issue would be much appreciated!

Thanks!

**Comment by TiagoSilvaPereira** [comment]
Hi @relayer464 please send me an email with your plugin code so I can test what is happening (there were no changes in Vemto in the last two months). As it seems not to be a bug with Vemto, I'll close for now

**Comment by relayer464** [comment]
Thanks, will do.
